### PR TITLE
directive: sort addition

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

**Comment by drjova** [comment]
merge first https://github.com/inveniosoftware/invenio-search-js/pull/34

**Comment by drjova** [comment]
@lnielsen rebased

File: example/index.html [hunk 1]
```diff
@@ -51,6 +51,23 @@
                    template="/src/invenio-search-js/templates/searchBar.html"
                    placeholder="Start typing">
                   </invenio-search-bar>
+                  <div class="text-right">
+                    <!-- SORT ORDER -->
+                    <invenio-search-sort-order
+                     label="Z->A"
+                     sort-key="sort"
+                     template="/src/invenio-search-js/templates/extras/toggleButton.html"
```
Comment (lnielsen) [inline example/index.html]: Not sure if I'm doing something wrong, but these templates seems to not be copied into ``dist`` folder when you do a build.

Comment (drjova) [inline example/index.html]: true, I flatten the template dir, it should be fine now.

**Pull request merged**